docs/docs/how-to-contribute.md

```diff
@@ -76,6 +76,7 @@ a pull request.
 
 - Clone the repo and navigate to `/www`
 - Run `yarn` to install all of the website's dependencies.
+- Create a `.env.development` file with a `GITHUB_API_TOKEN` environment variable. See [www/README][www-readme] for more info!
 - Run `gatsby develop` to preview the website in `http://localhost:8000`
```

Comment:
Change this to `yarn develop` so that it uses the local version of `gatsby-cli` instead. I was going to do a PR to fix this, but easier to do all at once. :)